proxies/live/apiproxy/policies/ServiceCallout.ValidateAccessToken.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ServiceCallout async="false" continueOnError="true" enabled="true" name="ServiceCallout.ValidateAccessToken">
+    <DisplayName>ServiceCallout.ValidateAccessToken</DisplayName>
+    <Properties/>
+    <Request clearPayload="true" variable="tokenValidationRequest">
+        <IgnoreUnresolvedVariables>false</IgnoreUnresolvedVariables>
```

Comment:
This step should fail if any of the referenced variables are not available. Otherwise it makes it hard to debug when things go wrong.